README.md

```diff
@@ -51,19 +52,28 @@
 
 * [Bidali](https://giftcards.bidali.com/buy-giftcards-with-dai/) - Buy gift cards with Dai.
 
-## Finance, Investment, and Trading
+## Finance, Investment
 
 * [Compound](https://compound.finance/) - Earn interest on Dai via algorithmic Money Markets.
 * [Dharma](https://www.dharma.io) - Lend your Dai and earn interest.
 * [Maker](https://cdp.makerdao.com/) - Get loans in Dai against collateralized ETH.
-* [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
+* [Nuo](https://app.nuo.network/) - Simplest way to margin trade.
+
+## Trading
```

Comment:
I don't hate the idea of breaking this out, but I'd like to adhere to alphabetical order, both for the ordering of sections and for the ordering of items within the section. Do you mind fixing that?

Reply:
Fixed in https://github.com/apbendi/use-dai/pull/2/commits/55bf547833a3795f45d711ec79fd8d62fdddbbeb